src/stylesheets/components/_concept.scss

```diff
@@ -281,3 +288,104 @@
     width: 0;
   }
 }
+
+.demo {
+
+  #msg-one {
+    opacity: 0;
+  }
+
+  #msg-two {
+    opacity: 0;
+  }
+
+  #msg-three {
+    opacity: 0;
+  }
+
+  #msg-four {
+    opacity: 0;
+  }
+
+  #msg-five {
+    opacity: 0;
+  }
+
+  #msg-six {
+    opacity: 0;
+  }
+
+  #msg-seven {
+    opacity: 0;
+  }
+
+  #msg-eight {
+    opacity: 0;
+  }
+}
+
+.messages-appearing.messages {
+
+  overflow: visible;
+  z-index: 2;
```

Comment:
Frank just told me about this: https://css-tricks.com/handling-z-index/

Might be a good idea to use that to mange your z-indexes so they don't get unwieldy